src/jobs/deploy.yml

```diff
@@ -2,22 +2,22 @@ description: >
   Deploy serverless applications.
 executor:
   name: default
-  machine-image: <<parameters.machine-image>>
+  machine_image: <<parameters.machine_image>>
 parameters:
-  machine-image:
+  machine_image:
     default: ubuntu-2204:2022.07.1
     type: string
     description: >
-      The CircleCI Linux Machine VM Image for this job. Find other available machine images here: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images
+      The CircleCI Linux Machine VM Image for this job. Find other available machine images here: https://circleci.com/docs/2.0/configuration-reference/#available-machine_images
   version:
     description: 'SAM CLI version to be installed.'
     type: string
     default: latest
   python_version:
-    description: 'If set, this version of Python will be installed and set with pyenv globally. ex: "3.7.0" This is only for the local environment and will not have any effect if use-container is enabled.9'
+    description: 'If set, this version of Python will be installed and set with pyenv globally. ex: "3.7.0" This is only for the local environment and will not have any effect if use_container is enabled.9'
```

Comment:
```suggestion
    description: 'If set, this version of Python will be installed and set with pyenv globally. ex: "3.7.0" This is only for the local environment and will not have any effect if use-container is enabled.'
```

Reply:
If it's really supposed to be `use_container` ignore the suggestion but remember to remove that `9` at the end.

Reply:
It's supposed to be `use_container`. I removed the `9`. Thank you!